VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -91,6 +96,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
             logger.error(ex);
         }
         loadData(getView());
+
+        String lastAccessedId = getArguments().getString(Router.EXTRA_LAST_ACCESSED_ID);
```

Comment:
The arguments bundle is initialized to a variable previously. If you remove the generic exception catching block, and take it out of the scope of the if condition, then it can be reused.